lib/Indexer/Tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php

```diff
@@ -159,6 +159,23 @@ public function provideFunctions(): Generator
      */
     public function provideMembers(): Generator
     {
+        yield 'show new object expressions when finding references on __construct except for superclass' => [
+            <<<'EOT'
+                // File: project/Bar.php
+                <?php class Bar { public function __construct() {} }
+                // File: project/subject.php
+                <?php class SubBar extends Bar { public function __c<>onstruct() {} }
+                // File: project/subject.php
+                <?php class Other extends Bar { public function __construct() {} }
+
+                // File: project/class1.php
+                <?php new Bar(); new SubBar(); new Other();
+                EOT
+                ,
+                2
```

Comment:
Only one `new Subbar()` is called.

Reply:
Yeah you are right. I think 2 is the number it currently returns.